### PR TITLE
Twilio SMS script

### LINKED ISSUE #36: Send sms using Twilio API with Python
# Description

Send SMS to people by creating a free Twilio account, use Python for scripting purpose.

## Type of issue
- [x] Feature (New Script)

## Checklist:

- [x] I have read the project guidelines.
- [x] I have checked all the existing projects, before submitting a new project issue.
- [x] I have checked previous issues to avoid duplicates.
- [x] This issue will be meaningful for the project.

### PR DESCRIPTION
**Pull request by Ajay-Raj-S**
# Description

Sending SMS to people by creating a free Twilio account.

Fixes #36 

## Type of change

- **`[X]`** New feature (non-breaking change which adds functionality)

# Checklist:

- **`[X]`** I have named my files and folder, according to this project's guidelines.
- **`[X]`** My code follows the style guidelines of this project.
- **`[X]`** My Pull Request has a descriptive title. (not a vague title like `Update index.md`)
- **`[X]`** I have commented on my code, particularly in hard-to-understand areas.
- **`[X]`** I have created a helpful and easy to understand `README.md`, according to the given [`README_TEMPLATE.`](https://github.com/Python-World/Python_and_the_Web/blob/master/README_TEMPLATE.md)
- **`[X]`** I have included a requirements.txt file (if external libraries are required.)
- **`[X]`** My changes do not produce any warnings.
- **`[X]`** I have added a working sample/screenshot of the script.

### CODE REVIEW

File: Scripts/twilio_sms/twilio_sms.py
Comment (AdityaJ7) [inline Scripts/twilio_sms/twilio_sms.py:37]: Add a function for this piece of code

**Review by AdityaJ7 — CHANGES_REQUESTED** [review]
Make functions for the later part of the code and you are good to go

**Comment by Ajay-Raj-S** [comment]
Hello @AdityaJ7 I will update it soon.

**Comment by Ajay-Raj-S** [comment]
Hello @AdityaJ7 , How to fix the check on DeepSource: error? Little help please

**Comment by AdityaJ7** [comment]
> Hello @AdityaJ7 , How to fix the check on DeepSource: error? Little help please

It will be fixed automatically don't worry about it

**Review by tusharnankani — APPROVED** [review]

**Comment by Ajay-Raj-S** [comment]
> > Hello @AdityaJ7 , How to fix the check on DeepSource: error? Little help please
> 
> It will be fixed automatically don't worry about it

Thank you, waiting for your approval.

**Review by AdityaJ7 — APPROVED** [review]

**Comment by AdityaJ7** [comment]
@all-contributors please add @Ajay-Raj-S for code

**Comment by Ajay-Raj-S** [comment]
Thank you @AdityaJ7.

**Pull request merged**
